code/modules/mining/abandoned_crates.dm

```diff
@@ -24,42 +24,41 @@
 	switch(loot)
 		if(1 to 5) //5% chance
 			new /obj/item/reagent_containers/food/drinks/bottle/rum(src)
-			new /obj/item/reagent_containers/food/snacks/grown/ambrosia/deus(src)
+			new /obj/item/reagent_containers/food/drinks/bottle/whiskey(src)
 			new /obj/item/reagent_containers/food/drinks/bottle/whiskey(src)
 			new /obj/item/lighter(src)
+			new /obj/item/reagent_containers/food/drinks/bottle/absinthe/premium(src)
+			new /obj/item/clothing/mask/cigarette/rollie/trippy(src)
+			new /obj/item/clothing/mask/cigarette/rollie/trippy(src)
+			new /obj/item/clothing/mask/cigarette/rollie/trippy(src)
 		if(6 to 10)
-			new /obj/item/bedsheet(src)
-			new /obj/item/kitchen/knife(src)
-			new /obj/item/wirecutters(src)
-			new /obj/item/screwdriver(src)
-			new /obj/item/weldingtool(src)
-			new /obj/item/hatchet(src)
-			new /obj/item/crowbar(src)
+			new /obj/item/melee/skateboard/pro(src)
```

Comment:
why remove the improv surgery kit

Reply:
cause it's useless probably

Reply:
it is entirely worthless and everyone knows it, besides it's not like you're gonna need it in fuckin' space

Reply:
if it's not code comments don't use the review feature for it reeeeeeeeeeeeee